utils/build.mjs

```diff
@@ -28,8 +28,8 @@ export const processActionYml = (
     actionYml,
     monorepoName,
 ) => {
-    const replacements = [];
-    Object.keys(packageJsons[name].dependencies).forEach((depName) => {
+    const replacements = [{from: new RegExp(`\\./actions/${name}/`), to: "./"}];
```

Comment:
Can you add a comment here explaining what this is for? Probably can just copy what you wrote in the PR description.